CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_google_ads_source v0.9.1
+
+## 🎉 Feature Enhancements 🎉
+- Updated macros, `get_ad_history_columns` and `get_campaign_history_columns`, to include `final_url_suffix` as a new column.
+- Added `final_url_suffix` into models, `stg_google_ads__campaign_history` and `stg_google_ads__ad_history` in order to gather all the UTM information for each campaign and ad because the UTMs can only be configured in this field when the campaign is set up.
```

Comment:
Do you by chance have Google Ads documentation that validates this statement? If so, it would be great to include in the CHANGELOG entry so we may provide more context to users.